fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1673,4 +1673,7 @@ public class Config extends ConfigBase {
 
     @ConfField(mutable = true)
     public static boolean enable_new_publish_mechanism = false;
+
+    @ConfField
+    public static int tablet_checker_interval_seconds = 60;
```

Comment:
So the check interval got extended to 60s, why?

Reply:
mistake, fixed